ts_src/issuance.ts

```diff
@@ -64,7 +65,17 @@ export function hashContract(contract: IssuanceContract): Buffer {
   if (!validateIssuanceContract(contract))
     throw new Error('Invalid asset contract');
 
-  return bcrypto.sha256(Buffer.from(JSON.stringify(contract)));
```

Comment:
what if `JSON.stringify(contract, undefined, 2)`

Reply:
and why not .sort on keys

Reply:
it works. But the members have to be sorted alphabetically so safer in my opinion to use a string template (Liquid dev website advice to use an external JS lib to stringify the contract but it sounds a bit overkill)

Reply:
We can't sort the keys using JSON.stringify: the solution is to use https://www.npmjs.com/package/json-stable-stringify